src/node_http_parser.cc

```diff
@@ -136,7 +136,10 @@ struct StringPtr {
 
   Local<String> ToString(Environment* env) const {
     if (str_)
-      return OneByteString(env->isolate(), str_, size_);
+      return String::NewFromUtf8(env->isolate(),
```

Comment:
Besides the question "is this more correct?", I am curious about the performance impact this may have.